src/test/java/server/EchoServerTest.java

```diff
@@ -1,12 +1,65 @@
 package server;
 
+
+import mocks.MockWrapper;
 import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
 import static org.junit.jupiter.api.Assertions.*;
 
 
-class EchoServerTest {
-    @Test void walkingSkeletonTest() {
-        EchoServer classUnderTest = new EchoServer();
-        assertEquals(classUnderTest.walkingSkeletonTest(), "testing 123");
+class MockWrappersTests {
```

Comment:
On your test class name here -- your file name is more accurate.  You're not testing the MockWrapper, you're testing the EchoServer.

Reply:
class name changes